### PR TITLE
add state/country for each city

### PR DESCRIPTION
**Pull request by learykara**

### CODE REVIEW

File: cities.yml [hunk 1]
```diff
@@ -1,239 +1,286 @@
...
+  state: FL
 
 - display_name: Greenville
   slug: greenville
+  state: SC
 
 - display_name: Hong Kong
   slug: hong-kong
   description: Hong Kong coding bootcamps help train people in one of the biggest commercial centers of East Asia. Learn to code in Hong Kong and dive into the tremendous financial industry, tech industry, or carve your own path.
+  state: null
```
Comment (diezguerra) [inline cities.yml]: https://www.youtube.com/watch?v=RDrfE9I8_hs

File: cities.yml [hunk 2]
```diff
@@ -1,239 +1,286 @@
...
   slug: little-rock
   description: In Arkansas, many people are learning to code in Little Rock. The coding bootcamps in Little Rock offer you an opportunity to get coding in the Midwest.
+  state: AR
 
 - display_name: London
   slug: london
   meta_description: London coding bootcamps are poised to launch you into technical work in London's financial or computing companies. Learn to code in London and get a head start on your career.
   description: At a global financial and trade hub, London coding bootcamps are in a prime position to help students find exciting work. Learn to code in London to seize this opportunity.
   featured: true
+  state: England
```
Comment (diezguerra) [inline cities.yml]: the sovereign state is the united kingdom 🤔

File: cities.yml [hunk 3]
```diff
@@ -1,239 +1,286 @@
...
   description: Las Vegas coding bootcamps can teach you the Microsoft stack, web development, data science, and more. Students who learn to code in Las Vegas have several creative industries to get involved in after graduating.
   featured: true
+  state: NV
 
 - display_name: Little Rock
   slug: little-rock
   description: In Arkansas, many people are learning to code in Little Rock. The coding bootcamps in Little Rock offer you an opportunity to get coding in the Midwest.
+  state: AR
 
 - display_name: London
```
Comment (diezguerra) [inline cities.yml]: https://www.youtube.com/watch?v=mDTPPuDw-BQ

File: cities.yml [hunk 4]
```diff
@@ -1,239 +1,286 @@
...
   slug: provo
   description: Provo is part of a growing Rocky Mountain tech scene. Join the industry by learning to code at a Provo coding bootcamp, not far from some of the country's biggest ski slopes.
+  state: UT
 
 - display_name: Raleigh
   slug: raleigh
   description: North Carolina is one of the Southeast's growing tech hubs and Raleigh is a big part. Learn to code at a Raleigh coding bootcamp to get involved in the web and mobile development industry.
+  state: NC
 
 - display_name: Raleigh-Durham
```
Comment (diezguerra) [inline cities.yml]: this is odd

File: cities.yml [hunk 5]
```diff
@@ -244,62 +291,75 @@
...
   slug: seattle
   meta_description: At a key tech hub for the Pacific northwest, Seattle coding bootcamps help students get jobs at major tech companies. Learn to code in Seattle to join the growing developer base.
   description: Seattle and its surrounding areas have a robust tech scene. Coding bootcamps in Seattle can help you get the skills to tap into this healthy market.
   featured: true
+  state: WA
 
 - display_name: Singapore
   slug: singapore
   description: As a major Asian marketplace, Singapore features coding bootcamps that can get you involved in global business.  Learn to code in Singapore and work in web or mobile development, or data science.
+  state: null
```
Comment (diezguerra) [inline cities.yml]: Singapore

File: cities.yml (hunk 1 above)
Comment (learykara) [inline cities.yml]: Wouldn't someone from hong kong not necessarily be thrilled if you said it was part of China?

File: cities.yml (hunk 2 above)
Comment (learykara) [inline cities.yml]: you're not wrong. fixing

File: cities.yml (hunk 4 above)
Comment (learykara) [inline cities.yml]: Having both Raleigh and Raleigh-Durham you mean? Yeah I thought so too, Raleigh-Durham is only a location for Thinkful. I figured there was some reasonable logic behind having both of them or we wouldn't have done it, but I'm happy to remove.

File: cities.yml (hunk 1 above)
Comment (diezguerra) [inline cities.yml]: Not a China expert, but the wikipedia says the state is China... 🤔

**Pull request merged**